File: erp/projects/report/employee_billing_summary/employee_billing_summary.py
# Copyright (c) 2013, CapKPI Technologies Pvt. Ltd. and contributors
# For license information, please see license.txt


import capkpi

from erp.projects.report.billing_summary import get_columns, get_data


def execute(filters=None):
	filters = capkpi._dict(filters or {})
	columns = get_columns()

	data = get_data(filters)
	return columns, data

File: erp/projects/report/employee_billing_summary/employee_billing_summary.js
// Copyright (c) 2016, CapKPI Technologies Pvt. Ltd. and contributors
// For license information, please see license.txt
/* eslint-disable */

capkpi.query_reports["Employee Billing Summary"] = {
	"filters": [
		{
			fieldname: "employee",
			label: __("Employee"),
			fieldtype: "Link",
			options: "Employee",
			reqd: 1
		},
		{
			fieldname:"from_date",
			label: __("From Date"),
			fieldtype: "Date",
			default: capkpi.datetime.add_months(capkpi.datetime.month_start(), -1),
			reqd: 1
		},
		{
			fieldname:"to_date",
			label: __("To Date"),
			fieldtype: "Date",
			default: capkpi.datetime.add_days(capkpi.datetime.month_start(), -1),
			reqd: 1
		},
	]
}
